eth/downloader/downloader.go

```diff
@@ -1274,9 +1280,13 @@ func (d *Downloader) fetchReceipts(from uint64, beaconMode bool) error {
 // queue until the stream ends or a failure occurs.
 func (d *Downloader) processHeaders(origin uint64, td, ttd *big.Int, beaconMode bool) error {
 	var (
-		mode       = d.getMode()
-		gotHeaders = false // Wait for batches of headers to process
+		mode        = d.getMode()
+		gotHeaders  = false // Wait for batches of headers to process
+		secondTimer = time.NewTimer(time.Second)
```

Comment:
```suggestion
		timer = time.NewTimer(time.Second)
```
I think this one is pretty self explanatory from the code.